tests/cellml_files/test_simple_odes.cellml

```diff
@@ -52,7 +52,7 @@ For annotations, we check:
       <unit units="second" prefix="micro"/>
    </units>
    <units name="mV">
-      <unit units="volt" prefix="milli"/>
+      <unit units="volt" prefix="-3"/>
```

Comment:
Just to trigger the part of the code in the parser test that deals with prefixes that are numbers it's also already covered elsewhere